### ISSUE TITLE
Negative parameter values

### ISSUE THREAD

**Issue by DimitrisVoul** [nlmixrdevelopment/nlmixr#176]
Hello,

Is there a way for nlmixr to give my negative parameter values? I want to fit a model to data to try and calculate different parameters but I would like for the model to be able to tell me whether their effect is positive or negative rather than me manually putting a minus sign in front of that parameter. 

Is something like that possible?

**Comment by mattfidler** [comment]
Indeed it is possible, simply don't exponentiate the parameter. Instead of

```r
model({
x <- exp(tx+eta.x)
})
```
you can use:

```r
model({
x <- tx + eta.x
})
```

**Comment by DimitrisVoul** [comment]
thank you very much. can that be:

tx*eta.x

?

**Comment by DimitrisVoul** [comment]
There is an issue when I try and to what suggested above. I will explain it with an example. I have 4 parameters 1 is positive and I know it and the rest I want to find whether their effect is negative or positive and I don't have a random effect in those. I tried 3 things:

1) parameters are initialised with positive values

parameter1<-exp(par1+eta.par1)
parameter2<-par2
parameter3<-par3
parameter4<-par4

This makes some parameters negative but at the end when trying to calculate the co-variance matrix it crashes.

2) parameters are initialised with positive values

parameter1<-par1+eta.par1
parameter2<-par2
parameter3<-par3
parameter4<-par4

This gives only positive values for par2-4 although i know that the effect of some of them must be negative. The confidence interval of these parameters includes zero.

3) parameters 2-4  are initialised with negative values.

parameter1<-par1+eta.par1
parameter2<-par2
parameter3<-par3
parameter4<-par4

This makes the fitting stop immediately at the initial values I gave.
